improver/calibration/ensemble_calibration.py

```diff
@@ -994,6 +993,8 @@ def compute_initial_guess(
             List of coefficients to be used as initial guess.
             Order of coefficients is [alpha, beta, gamma, delta].
         """
+        import statsmodels.api as sm
```

Comment:
Why is this import statement not at the top of the file with all the other import statements?

Reply:
As mentioned above, this is because statsmodels is being treated as optional for the codebase, even though it is required for EMOS. Therefore, if someone isn't using the EMOS component of improver, then they can use a slimmer environment.